e2e/Microsoft.Azure.Devices.E2ETests/TwinE2ETests.cs

```diff
@@ -487,11 +487,9 @@ static public void ClassCleanup()
 
             Tuple<string, string> deviceInfo = TestUtil.CreateDevice(DevicePrefix, hostName, registryManager);
             var deviceClient = DeviceClient.CreateFromConnectionString(deviceInfo.Item2, transport);
-
-// TODO: #193
```

Comment:
Same as above: since you didn't remove the pragma, please keep the TODO as #193 tracks the obsolete calls.